locales/cy.json

```diff
@@ -53,52 +53,52 @@
     "generic": "Rhowch %s dilys",
     "required": "Ni all y maes hwn fod yn wag",
     "currency": "Dewiswch swm mewn punnoedd a cheiniogau drwy ddefnyddio digidau a phwynt degol. Er enghraifft “10.50”",
-    "isAboveMaxAmount": "Dewiswch swm sy'n llai na £%s",
-    "isGreaterThanMaxLengthChars": "Mae'r testun yn rhy hir. Ni all fod yn fwy na 50 o nodau.",
-    "invalidCharacters": "Ni allwch ddefnyddio unrhyw rai o'r nodau canlynol < > ; : ` ( ) \" ' = &#124; \",\" ~ [ ]",
+    "isAboveMaxAmount": "Dewiswch swm sy’n llai na £%s",
+    "isGreaterThanMaxLengthChars": "Mae’r testun yn rhy hir. Ni all fod yn fwy na 50 o nodau.",
+    "invalidCharacters": "Ni allwch ddefnyddio unrhyw rai o’r nodau canlynol < > ; : ` ( ) \" ’ = &#124; \",\" ~ [ ]",
```

Comment:
I think in the list of invalid characters we do actually want a straight `'` (the earlier one is fine though)

Reply:
Good spot Alex, will change it.